Add link to Backend layout page (#53)
TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage: Documentation/ContentMapping/Index.rst

images, etc. to areas in the frontend. This can be achieved easily by using
custom TypoScript.

> More information about setting up various columns and rows can be found at the
> page :ref:`Backend layout <t3coreapi:be-layout>`.

Reviewer: ```suggestion
See :ref:`Backend layouts <t3coreapi:be-layout>` for more information about setting up various columns and rows.
```
